src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/LocationFiltersForm.test.tsx

```diff
@@ -628,4 +660,52 @@ describe('LocationFiltersForm', () => {
       expect(handleSubmit).toHaveBeenCalledWith(expected);
     });
   });
+
+  test('sets the step heading to `Choose locations` if multiple location types', () => {
+    render(
+      <LocationFiltersForm
+        {...testWizardStepProps}
+        options={testLocationsFlat}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(
+      screen.getByRole('heading', {
+        name: 'Step 1 (current) Choose locations',
+      }),
+    ).toBeInTheDocument();
+  });
+
+  test('sets the step heading based on location type if only one type', () => {
+    render(
+      <LocationFiltersForm
+        {...testWizardStepProps}
+        options={testSingleLocationType}
```

Comment:
For these one-off test variables, I'd suggest inlining them into the test case. Would only use suite scope variables if they're being used in multiple places to avoid having to jump around the file unnecessarily.

Reply:
Done